pages/en/advanced/mina-signer.mdx

```diff
@@ -1,11 +1,11 @@
 import Page from "@reason/pages/Docs";
 import DocLink from "@reason/components/DocLink";
-export default Page({ title: "Mina Client SDK Keypair" });
+export default Page({ title: "Mina Signer" });
 
-# Client SDK
+# Mina Signer
 
-This is a NodeJS client SDK that allows you to generate Mina key pairs and sign transactions and strings using Mina's key pairs.
-The project contains Typescript and ReasonML typings but can be used from plain NodeJS as well.
+This is a NodeJS SDK that allows you to generate Mina key pairs and sign transactions and strings using Mina's key pairs.
```

Comment:
`Mina Signer is a NodeJS package`...  I think we should use plainer language and avoid calling it `SDK`